src/enums/componentsCompanyInfo.ts

```diff
@@ -8,6 +8,7 @@ export enum ComponentsCompanyInfo {
   'company-name-section',
   'change-business-type',
   'name-translation',
+  'business-start-date',
```

Comment:
As this object is used for various entity types, please test those other entity types (manually is OK) with respect to this component's validity state.

Reply:
Okay I will test this with the other entity types that use this locally and then add that they have been confirmed to the PR

Reply:
Awesome, perfect!